.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E402,E731,D100,D101,D102,D103,D104,D105,W503,W504,E252
+ignore = E402,E731,D100,D101,D102,D103,D104,D105,W503,W504,E252,B006
```

Comment:
I wouldn't ignore it -- in general mutable defaults isn't the best practice. 

In this specific case those parameters aren't used anymore and should just be removed.